public/data/glTF-projects-data.json

```diff
@@ -2160,5 +2146,51 @@
   "type" : [ "application" ],
   "language" : [ "c++" ],
   "inputs" : [ "glTF 2.0" ]
+}, {
+  "name": "Cesium for Unreal",
+  "link": "https://cesium.com/platform/cesium-for-unreal/",
+  "description": "A plugin for the Unreal Engine that allows streaming assets from the cloud, using the 3D Tiles format that is based on glTF",
+  "task": ["view"],
+  "type": ["plugin"],
+  "license": ["Apache-2.0"],
+  "language": ["C++"],
+  "inputs": ["3D Tiles", "glTF 2.0"]
+}, {
+  "name": "Cesium Native",
+  "link": "https://github.com/CesiumGS/cesium-native",
+  "description": "A base library for 3D geospatial applications, including lightweight glTF serialization and deserialization",
+  "task": ["load", "import", "export"],
+  "type": ["library"],
+  "license": ["Apache-2.0"],
+  "language": ["C++"],
+  "inputs": ["glTF 2.0"],
+  "outputs": ["glTF 2.0"]
+}, {
+  "name": "CesiumJS",
+  "link": "https://cesium.com/platform/cesiumjs/",
+  "description": "A library for creating geospatial web applications, and visualize data in the 3D Tiles format that is based on glTF",
```

Comment:
CesiumJS can also load glTF directly.  In the context of glTF Explorer, that is probably even more important than mentioning 3D Tiles.